cherrypy/tutorial/README.rst

```diff
@@ -14,3 +14,24 @@ applications using CherryPy. A couple of notes:
   variable substitution. In most real-world applications, you will
   probably want to use a separate template package (like Cheetah,
   CherryTemplate or XML/XSL).
+
```

Comment:
Should this README be somehow included @ https://github.com/cherrypy/cherrypy/blob/master/docs/tutorials.rst ?

Reply:
Yes, maybe only the new pats, since the file you refer already has the other information, not related to docker. Also the parts I added for docker, assume that the cherrypy/cherrypy image exists in hub.docker.com, that is why I didn't provide build instructions for the image.

Reply:
Well, that page is what's shown @ https://docs.cherrypy.org/en/latest/tutorials.html.
I'm just worried that only those who look into the repo contents will discover that the docker image exists.

Reply:
I can either duplicate the text over there or add only a reference. In the first case, that could be right below the introduction paragraph.

Since the goal of the Tutorials section in the docs is teaching by example and running the tutorials within docker won't give access direct access to the code, I can point to the existence of the tutorials in the code.

Reply:
Right, or add another docs page and `include` this file there.
FWIW you can even include RST files partially with Sphinx.

Reply:
I'll take a look at it later today